### PR TITLE
feat: add medusa module

### PR DESCRIPTION
**Pull request by Intevel**
https://github.com/Baroshem/nuxt-medusa

@Baroshem

### CODE REVIEW

File: modules/medusa.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: medusa
+description: Easily connect to Medusa from your Nuxt 3 application
+repo: Baroshem/nuxt-medusa
+npm: nuxt-medusa
+icon: medusa.svg
+github: https://github.com/Baroshem/nuxt-medusa/
+website: https://nuxt-medusa.vercel.app/
+learn_more: https://nuxt-medusa.vercel.app/
+category: CMS
```
Comment (Baroshem) [inline modules/medusa.yml:9]: suggestion: Maybe we can create a new filter for e-commerce? I am sure there is a module for Shopify and Swell :)

**Review by Baroshem — COMMENTED** [review]
One suggestion from my side. Thanks for this PR 💚

File: modules/medusa.yml (hunk 1 above)
Comment (Intevel) [reply modules/medusa.yml:9]: Yes I was already strugglen which category Medusa is. Adding an e-commerce Category would be nice 👯

Comment (Intevel) [reply modules/medusa.yml:9]: @Baroshem https://github.com/nuxt/modules/blob/main/lib/categories.ts#L10 There is actually an E-Commerce Category

**Review by Baroshem — APPROVED** [review]

**Review by danielroe — APPROVED** [review]

**Pull request merged**